x/blob/types/payforblob.go

```diff
@@ -165,7 +164,7 @@ func EstimateGas(blobSizes []uint32, gasPerByte uint32, txSizeCost uint64) uint6
 // DefaultEstimateGas runs EstimateGas with the system defaults. The network may change these values
 // through governance, thus this function should predominantly be used in testing.
```

Comment:
The network can't change the params via governance any more so this comment looks incorrect.

```suggestion
// DefaultEstimateGas runs EstimateGas with the system defaults. 
```